src/ServiceControl.Config/App.xaml.cs

```diff
@@ -16,6 +17,23 @@ public App()
             {
                 ExceptionMessageBox.Attach();
             }
+
+            EnsureStandardPopupAlignment();
+        }
+
+        static void EnsureStandardPopupAlignment()
```

Comment:
A bit nitpicking here, but could you put the logic in a class and use it in App? This is a workaround and clutters the App.cs code. e.g.

```StandardPopup.ApplyAlignment();```

The  hint/comment can also be moved into the class implementing the workaround.

Reply:
@HEskandari moved.